checker/manual/external-tools.tex

```diff
@@ -792,6 +793,24 @@ \section{tIDE\label{tide}}
 % See its documentation at \myurl{http://tide.olympe.in/}.
 \end{sloppypar}
 
+\section{NetBeans\label{netbeans}}
+To run a checker within NetBeans:
+
+\begin{enumerate}
+\item Add the checker framework libraries to your project's library. To do so, 
+right click on the project in the Projects panel, and select "Properties" in the
```

Comment:
Since this is a LaTeX document, please use LaTeX-style quotes throughout.  The open-quote is written as two left single quotes
```
``
```
and the close-quote is written as two right quotes
```
''
```
.